README.md

```diff
@@ -28,24 +28,105 @@ Please consider donating to cover up our hosting costs. 🙏
 
 ## Development
 
-1. Clone this repository by running
+Clone this repository by running
+```sh
+git clone https://github.com/swapagarwal/swag-for-dev.git
+```
+
+### Local
+
+1. Ensure you are running an up-to-date version of [Node.js](https://nodejs.org/en/download/package-manager/) on your machine, and that you have npm installed.
+1. Open a terminal in the `swag-for-dev` directory. Type
+    ```sh
+    npm install
     ```
-    git clone https://github.com/swapagarwal/swag-for-dev.git
+    into the terminal to install the dependencies. To start the webserver, type
+    ```sh
+    npm start
     ```
-1. Ensure you are running an up-to-date version of [Node.js](https://nodejs.org/en/download/package-manager/) on your machine, and that you have npm installed.
-1. Make sure you have [Gulp](https://gulpjs.org/getting-started.html) installed by running
+1. The website should open in a browser after it compiles, or you can view it by going to http://localhost:8000
+
+### Docker
```

Comment:
I think maybe we should move Docker instructions to a separate place? It kind of breaks the flow. Maybe add a `README-Docker.md`?

Reply:
Maybe it could go to https://github.com/swapagarwal/swag-for-dev/blob/master/CONTRIBUTING.md with the whole local setup? or we can create a `GETTING_STARTED.md` or `docs/getting_started.md`? WDYT?

Reply:
Sure 👍 I definitely think we should break up the docs a bit to make them a bit easier to browse

Reply:
Can add a **Table of contents**

Reply:
Moved it to CONTRIBUTING.md